docs/lab/overview/quickstart.mdx

```diff
@@ -0,0 +1,285 @@
+
+In the following, replace `{{BASE_URL}}` with your team's [environment URL](/home/environments).
+
+### (1) Creating a user
+
+In the context of our API, a user refers to the patient who will be undergoing the lab test. All you need to provide is a string identifying the user on your side (`client_user_id`). The response to this call returns the user ID on our side, which you'll be able to use for future requests.
+
+You can find more details in our [API quickstart guide](/home/quickstart#3-creating-your-first-user).
+
+<SnippetGroup>
+
+```bash Creating a Vital user (bash)
+curl --request POST \
+     --url {{BASE_URL}}/v2/user/ \
+     --header 'Accept: application/json' \
+     --header 'Content-Type: application/json' \
+     --header 'x-vital-api-key: <YOUR_API_KEY>' \
+     --data '{"client_user_id":"<YOUR_CLIENT_USER_ID>"}'
+```
+
+```js Node
+const user = await client.User.create(client_user_id);
+```
+
+```python Python
+user = client.User.create(client_user_id)
+```
+
+</SnippetGroup>
+
+### (2) Listing Available Tests
+
+To retrieve the set of lab tests you have access to, simply use the `/v3/lab_tests` API endpoint. In Sandbox, you will already have access to a default set of tests, to simplify the initial integration. Once you're ready to head to production, we can [setup a call](/lab/overview/introduction#production-launch) to organize everything and get you ready for launch!
+
+<SnippetGroup>
+
+```bash Listing available tests (bash)
+curl --request GET \
+     --url {{BASE_URL}}/v3/lab_tests/ \
+     --header 'Accept: application/json' \
+     --header 'x-vital-api-key: <YOUR_API_KEY>' \
+     --header 'Content-Type: application/json'
+```
+
+```js Node
+const tests = await client.LabTests.getTests();
+```
+
+```python Python
+tests = await client.LabTests.get_tests();
+```
+
+</SnippetGroup>
+
+This will return a list of all the available tests to you. They include a description, turnaround time and price:
+
+```json Response
+{
+  "lab_tests": [
+    {
+      "id": "e2eaa385-a311-4f17-b33f-2165e3d24dd9"
+      "name": "Lipids Panel",
+      "description": "Cholesterol tests",
+      "sample_type": "dried blood spot",
+      "method": "testkit",
+      "price": 10.0,
+      "is_active": True,
+      "lab": {
+          "slug": "USSL",
+          "name": "US Specialty Lab",
+          "first_line_address": "123 Main St",
+          "city": "New York",
+          "zipcode": "10001",
+      },
+      "markers": [
+          {
+              "name": "Thyroid Stimulating Hormone",
+              "slug": "tsh",
+              "description": "",
+              "min_value": 100,
+              "max_value": 200,
+              "unit": "fg/L"
+          }
+      ],
+    }
+  ]
+}
+```
+
+You can then use the test `id` when creating an order.
+
+### (3) Placing an order
+
+Ordering a test for your user, is as simple as making an API call:
```

Comment:
the comma here doesn't work. The easiest way to see if a comma makes sense is to flip the two parts:

<part 1> comma <part 2>

<part 2> comma <part 1>

If they both read well, then a comma can make sense, but in this case it doesn't:

> is as simple as making an API call, ordering a test for your user

Reply:
That's a great comma test. Didn't know about it!